projects/components/src/multi-select/multi-select.component.test.ts

```diff
@@ -91,8 +93,10 @@ describe('Multi Select Component', () => {
     });
 
     spectator.tick();
-    const selectedElements = spectator.queryAll('input:checked', { root: true });
-    expect(selectedElements.length).toBe(2);
+    const selectedElements = spectator.queryAll('ht-checkbox', { root: true });
+    expect(
+      selectedElements.filter(checkboxOption => checkboxOption.getAttribute('ng-reflect-checked') === 'true').length
```

Comment:
@palbizu I think you are forced to use this function since queryAll does not support using the directive `CheckboxComponent` with the root option.  I know you want to test whether `checked` property is set to true or not. 

Mmm, may be we can just test whether `selectedElements` (better rename it to checkboxes) length ===2. No need to test checked property in this case, since using a public function for a child component is even worse.

Reply:
Ah, I didn't realize that wasn't supported - what about `spectator.queryAll('ht-checkbox', { root: true, read: CheckboxComponent })` ?

Reply:
@aaron-steinfeld your approach is not supported

Reply:
ah, sounds like a good improvement to spectator :) without that, if we don't need to verify the checked property for correctness, we can probably drop it - but somewhere, we should have a test verifying that the checkboxes are set correctly. As much as I dislike it, it's better than not testing it at all if we don't have a better way.

Reply:
Another option is to use the real checkbox instead of mocking it, which would allow you to query its contents - a tradeoff of bad options though.